src/IKVM.Runtime/Java/Externs/java/lang/Class.cs

```diff
@@ -46,7 +47,12 @@ static class Class
                 global::java.lang.SecurityManager sm = global::java.lang.System.getSecurityManager();
                 if (sm != null)
                     sm.checkPermission(new global::java.lang.RuntimePermission("createClassLoader"));
-                Type type = Type.GetType(name);
+                Type type = null;
+                foreach (Assembly asm in AppDomain.CurrentDomain.GetAssemblies())
```

Comment:
This seems very much wrong. It is avoiding the entire class load hierarchy, and scanning all AppDomain assemblies loaded for every class lookup? What if the hierarchy isn't configured to do so?

Also, on Core, this only uses the default assembly load context, and not any nested ones.